eris-platform/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "itemDisplayName": "Eris Platform Node",
+  "description": "Eris: the permissionable, smart contract focused blockchain application platform",
+  "summary": "This template deploys Eris on a Ubuntu node within Azure",
+  "githubUsername": "eris-ltd",
```

Comment:
i won't be maintaining this file forever. it will be collective. prefer to keep it as is. if it will block I'll change it.